plugins/modules/database/misc/odbc.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, John Westcott <john.westcott.iv@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: odbc
+author: "John Westcott IV (@john-westcott-iv)"
+version_added: "2.8"
+short_description: Execute SQL via ODBC
+description:
+    - Read/Write info via ODBC
+options:
+    dsn:
+      description:
+        - The connection string passed into ODBC
+      required: True
```

Comment:
```suggestion
      required: true
```
Either use `true`/`false` or `yes`/`no` in docs.